pkg/ipamd/ipamd.go

```diff
@@ -1741,39 +1683,39 @@ func getMinimumIPTarget() int {
 }
 
 func disableENIProvisioning() bool {
-	return getEnvBoolWithDefault(envDisableENIProvisioning, false)
```

Comment:
nit: let's delete `getEnvBoolWithDefault` in `pkg/ipamd/introspect.go` since all the uses are being deleted